libraries/AP_Scripting/applets/Aerobatics/FixedWing/RateBased/README.md

```diff
@@ -0,0 +1,144 @@
+# Sport Aerobatics
+
+The lua script "sport_aerobatics.lua" implements scripted aerobatics,
+allowing fixed wing aircraft to execute a number of aerobatic
+manoeuvres either in AUTO mission or by triggering using pilot commands
+using RC switches.
+
+As always, but particularly with scripted aerobatics, test in SITL until 
+you understand the function and behaviour of each manouver. You will need 
+an appropriate aircraft, and be ready to take manual control if necessary!
+
+These tricks are rate-based vs the precision trajectory-based manouvers of the script in the TrajectoryBased directory above this one. However, many planes have difficulty completing those tricks unless they can sustain extended vertical climbs and knife-edges, especially in wind. These tricks do not try to maintain a geospatial track, but rather, just attitude rates. Even planes that cannot hold the 90 deg knife edge trick, will probably hold the 180 deg (inverted) one and do loops and rolls, and these tricks can be done in even strong wind, although will not be well shaped.
+
+## Available Manoeuvres
+
+The following table gives the available manoeuvres. Each manoeuvre has
+an ID number which is used in the AUTO mission or in the TRIKn_ID
+parameters (described below). The present ground track is used as the track for the trick.
+The "Turnaround" column indicates if the manoeuvre results in a course reversal, which impacts how it is used in 
+AUTO missions. Once the trick is completed, the mode that was being used at the start of the trick is restored. If the mode is CRUISE, its
+track and altitude are reset to the values present when the mode is restored. Tricks in AUTO missions require that they be performed between two waypoints to establish
+the ground track.
+
+| ID | Name                     | Arg1            | Arg2                       | Turnaround |
+| -- | ------------------------ | ------------_-- | -------------------------- | -----------|
+| 1  | Roll(s)                  | rollrate(dps)   | num rolls                  | No         |
+| 2  | Loop(s)/TurnAround       | pitchrate(dps)  | num loops or turnaround(0) | if num=0   |
+| 3  | Rolling Circle           | yawrate(dps)    | width                      | No         |
+| 4  | KnifeEdge                | roll angle(deg) | length(sec)                | No         |
+| 5  | Pause                    | length(sec)     | na                         | No         |
+
+
+## Loading the script
+
+Put the plane_aerobatics.lua script on your microSD card in the
```

Comment:
Should this say sport_aerobatics.lua? (Seems like it should be the same script referred to at line 3)

Reply:
yep....I took the trajectory  readme and hacked at it...missed that one though